docs/custom_gates.ipynb

```diff
@@ -552,7 +552,7 @@
     "These gates can be understood by the simulator, optimizers, and other code.\n",
     "3. All that matters is functional equivalence.\n",
     "Don't worry about staying within or reaching a particular gate set; it's too hard to predict what the caller will want. Gate-set-aware decomposition is useful, but *this is not the protocol that does that*.\n",
-    "Gate-set-aware decomposition may be added in the future, but doesn't exist within Cirq at the moment.\n",
```

Comment:
We need to update documentation of _decompose_ protocol and mention that decompose will now always try to end at default cirq gateset, i.e. x/y/z/cz/measurement/global phase